src/subdomains/supporting/pricing/domain/entities/asset-price.entity.ts

```diff
@@ -0,0 +1,18 @@
+import { Asset } from 'src/shared/models/asset/asset.entity';
+import { IEntity } from 'src/shared/models/entity';
+import { Column, Entity, ManyToOne } from 'typeorm';
+
+@Entity()
+export class AssetPrice extends IEntity {
+  @ManyToOne(() => Asset, (a) => a.assetPrices)
+  asset: Asset;
+
+  @Column({ type: 'float', nullable: false })
+  eurPrice: number;
+
+  @Column({ type: 'float', nullable: false })
+  usdPrice: number;
+
+  @Column({ type: 'float', nullable: false })
+  chfPrice: number;
```

Comment:
I think I would use `priceEur`, `priceUsd` and `priceChf` (that's what we used in other places)